plugins/inputs/mqtt_consumer/README.md

```diff
@@ -100,6 +100,11 @@ to use them.
   ## Use TLS but skip chain & host verification
   # insecure_skip_verify = false
 
+  ## Client trace messages
+  ## When set to true, the MQTT client's messages are printed to stdout. These
+  ## messages are very noisey, but essential for debugging issues.
```

Comment:
Can you please mention here that this only has an effect if Telegraf is run in debug mode!?

Reply:
done!